### ISSUE TITLE
Typst 0.12 is out :)

### ISSUE THREAD

**Issue by aik2mlj** [fenjalien/obsidian-typst#67]
Hi, I appreciate this plugin a lot and it really boost the productivity taking math notes.
Would you consider updating the typst binary to 0.12 since it's out recently?

**Comment by fenjalien** [comment]
Yes I am aware, I've been busy with other work. I'm starting to look into it now though.

**Comment by aik2mlj** [comment]
A gentle bump to the update. Seems that the latest commit hasn't been made into a new release?

**Comment by fenjalien** [comment]
Apologies for the continued silence, 0.12 is working on the latest commit but iirc something had broken with file loading so packages don't work. 

Unfortunately I'm having to put all projects on hiatus until end of May, sorry again for any inconveniences caused.

**Comment by zzeitt** [comment]
And there is Typst 0.13 now, which support html export: https://typst.app/blog/2025/typst-0.13#a-first-look-at-html-export.
